components/elm/src/biogeophys/HydrologyDrainageMod.F90

```diff
@@ -48,14 +48,14 @@ subroutine HydrologyDrainage(bounds,    &
     ! !USES:
     use landunit_varcon  , only : istice, istwet, istsoil, istice_mec, istcrop
     use column_varcon    , only : icol_roof, icol_road_imperv, icol_road_perv, icol_sunwall, icol_shadewall
-    use clm_varcon       , only : denh2o, denice, secspday
-    use clm_varctl       , only : glc_snow_persistence_max_days, use_vichydro, use_betr
+    use elm_varcon       , only : denh2o, denice, secspday
+    use elm_varctl       , only : glc_snow_persistence_max_days, use_vichydro, use_betr
     use domainMod        , only : ldomain
     use atm2lndType      , only : atm2lnd_type
-    use clm_varpar       , only : nlevgrnd, nlevurb, nlevsoi    
+    use elm_varpar       , only : nlevgrnd, nlevurb, nlevsoi    
     use clm_time_manager , only : get_step_size, get_nstep
     use SoilHydrologyMod , only : CLMVICMap, Drainage
-    use clm_varctl       , only : use_vsfm
+    use elm_varctl       , only : use_vsfm
     use BeTRSimulationALM, only : betr_simulation_alm_type
```

Comment:
Should this be named to betr_simulation_elm_type?

Reply:
@tanzeli1982, `BeTRSimulationALM` is part of BeTR and @jinyuntang will have to make a change in BeTR repo.